contentctl/input/director.py

```diff
@@ -221,6 +221,13 @@ def constructDetection(self, builder: DetectionBuilder, file_path: str) -> None:
         # TODO: is there a better way to handle this? The `test` portion of the config is not defined for validate
         if (self.input_dto.config.test is not None) and (not self.input_dto.config.test.enable_integration_testing):
             builder.skipIntegrationTests()
+        
+        if builder.security_content_obj is not None and \
```

Comment:
This is less about this MR and more so about the class hierarchy of SecurityContentObject

Why does the DetectionBuilder class have an attribute for the SecurityContentObject and not just for a Detection?

Reply:
💯 
There are huge changes/simplifications coming to the Builders in the following branch:
https://github.com/splunk/contentctl/tree/pydantic2
I am resolving this conversation because we will begin to address them there!